### DISCUSSION TITLE
Suggested Lists Disappear After Creating First List

### DISCUSSION THREAD

**Discussion by Maggard** [community/community#68848] [Other Feature Feedback, Questions, & Ideas]
When adding the first project to a list, the selection pop-up shows:  
 · Three suggested lists
 · "Create List" link to add new List

When adding the next List,  the selection pop-up shows ONLY the custom List created above. 

Suggestion
￣￣￣￣￣ 
Continue to suggest the three lists until:
 · A project is added to a suggested list which moves the suggestion to the available list.
 · User deletes the suggested list. 
 
If all lists are deleted by a User, the three suggested lists again display when adding projects.

**Comment by 1amkaiz3n** [comment]
It seems like you're suggesting a user interface enhancement for a feature where users can organize projects into lists. The suggestion is to keep displaying the three suggested lists and the option to create a new list until the user adds a project to a suggested list (at which point it moves to the available list), or the user deletes the suggested list, reverting back to the initial display of suggested lists.

Here's a possible flow based on your suggestion:

1.When adding a project for the first time:

  - Display three suggested lists.
  - Provide an option to "Create List" to add a new custom list.

2.When adding the next project:

- Display the custom list created in the previous step.
- Continue to display the three suggested lists and the "Create List" option.

3.If a project is added to a suggested list:

- Move that list from the suggested list to the available list.

4.If a user deletes a suggested list:

- Remove the list from the available list.
- If all lists are deleted by the user, revert back to displaying the three suggested lists and the "Create List" option.


This approach ensures that suggested lists and the option to create new lists are always available until the user interacts with them (by adding a project or deleting a list), promoting a user-friendly experience and flexibility in managing projects and lists.
